docs/development/deploying-at-scale/index.md

```diff
@@ -1,3 +1,41 @@
 # Deploying at scale
 
 REANA can be easily deployed on large Kubernetes clusters using Helm. Useful for production instances.
+
+## Pre-requisites
+
+REANA needs a shared file system to host all analyses workspaces when running in a multinode deployment setup. Therefore, you should create an [`StorageClass`](https://kubernetes.io/docs/concepts/storage/storage-classes/#the-storageclass-resource) pointing to your storage backend. The `StorageClass` should meet the following requirements:
+
+- Be named `<helm-release-prefix>-shared-volume-storage-class`
+- Be created in the same namespace as the one you will deploy REANA to
```

Comment:
Mention that the default namespace is `reana`.

Reply:
If you mean Kubernetes namespace, we actually deploy everything in `default`, we didn't put work on changing namespace (runtime pods need work to support configurable namespace).

Reply:
Ah, yes, indeed.  We can address this once we have different namespace for infrastructure pods and different namespace for runtime user workflow pods and jobs.